jgrapht-core/src/main/java/org/jgrapht/GraphPath.java

```diff
@@ -95,6 +124,15 @@
      * @return the weight of the path
      */
     double getWeight();
+
+    /**
+     * Returns the length of the path, measured in the number of vertices.
+     * @return the length of the path, measured in the number of vertices
+     */
+    default int getLength(){
```

Comment:
This method name is confusing since usually path length refers to edges...is there a more specific name we can use?